build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -54,7 +54,12 @@
     wait_for_iree_benchmark_module_start,
     parse_iree_benchmark_metrics,
 )
-from common.benchmark_suite import MODEL_FLAGFILE_NAME, BenchmarkCase, BenchmarkSuite
+from common.benchmark_suite import (
+    MODEL_FLAGFILE_NAME,
+    BenchmarkCase,
+    BenchmarkSuite,
+    get_run_configs_by_target_and_shard,
```

Comment:
nit: Style guide suggests not to import class, function but only import modules to avoid naming collision (https://google.github.io/styleguide/pyguide.html#22-imports)

We are in process to refactor the existing code to follow that style. So no need to touch the existing code, but I hope we follow the new style for the new code.

Reply:
Good point. Done.